scripts/tcpperf_server.py

```diff
@@ -22,14 +22,14 @@ def __init__(self, pause=60, values=1, start_port=8123, processes=2):
         self.port = start_port
         self.processes = processes
 
-    def client(self, n=2000, format_='hello'):
+    def client(self, n_=2000, format_='hello'):
```

Comment:
why `n_`?

Reply:
n was shadowing n from the outer scope. Would it be better if I change this to "amount_of_connections"?

Reply:
Would it be better to have a main function to wrap what is currently in the `if __name__ == '__main__'` block? There are quite a lot of variable added to the global scope there indeed

Reply:
Done. It's a better solution.